speaker-notes.js

```diff
@@ -230,7 +230,8 @@
       markDefunct();
       break;
     default:
-      if (window.location.pathname == "/print.html") {
+      const printPageRegexp = /print\.html?/;
+      if (window.location.pathname.match(printPageRegexp)) {
```

Comment:
Ah, thanks for finding the problem so quickly!

Since we want to check if the string [ends with](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith) `/print.html`, could we do this instead:

```suggestion
      if (window.location.pathname.endsWith("/print.html")) {
```

This way we won't get a false negative on a page named `ups-print.html`.

Reply:
My biggest concern would be for instance `webassembly/print.html`

Reply:
> Sounds good, I wasn't aware of that method

Yeah, I didn't know about it either :smile: 

> My biggest concern would be for instance `webassembly/print.html`

Right, but that should only happen if we make a `print.md` file ourselves — and in that case we will get an error from mdbook.